### DISCUSSION TITLE
Planning for next phase

### DISCUSSION THREAD

**Discussion by niceit** [niceit/tabby-cloud-sync-settings#16] [Ideas]
- Google Cloud Storage.
- Gists for Gitee, Bitbucket.
- Minio S3 alternative.
- Saving password, key file vault??... Good or bad security?
- Revision of setting changes??
- Multiple saved settings for cloud?? Now it just has one config at a time.
- Rich view for syncing log?? (currently saved to files)

**Comment by Luluno01** [comment]
Hi, I got some ideas. Hope they would help.

* SharePoint/OneDrive support.
* Any (private) Git repository support (inherent revision tracking).
* Multiple saved and activated backend settings (for fault tolerance).

Also, do you have any guidance shall one wants to contribute by implementing a new backend? Thanks!

**Comment by niceit** [comment]
thank @Luluno01 
By the limited of time, I will sort out and update the planned things. Really appreciated your ideas.
